### ISSUE TITLE
Shouldn't the number of images presented be fixed?

### ISSUE THREAD

**Issue by danielbicho** [arquivo/pwa-technologies#653]
The number of images that Image Search is presenting is a little 'random', what is the criteria?
Shouldn't it display a fixed number of images?

Like the full text search that always displays the first 10 results.

**Comment by igobranco** [comment]
I've confirmed this issue.

After first analisis of it we've detected that it's suppose to display always 25 images.

On the [user interface searching by images using 'ronaldo'](https://m.arquivo.pt/images.jsp?query=ronaldo&l=pt&dateStart=01%2F01%2F1996&dateEnd=31%2F12%2F2018
) term. Currently is displayed only 20 images. The count have been made on chrome console using javascript and xpath:
```javascript
$x("//*[@class=\"imageContent\"]").length
```
Result:
```
20
```

When opening the network tab on chrome developer tools, it was querying the Arquivo.pt Image Search API, but the result show other problem. The API call only returns 23 items when it's returning a 31K total items.
A simplification of [Arquivo.pt Image Search API call](https://arquivo.pt/imagesearch?q=ronaldo&from=19960101000000&to=20181231235959&offset=0&maxItems=25&more=imgThumbnailBase64)
It returns on first results page:
```
"totalItems": 31429,
"numberOfResponseItems": 23,
```
